.github/workflows/ci-test.yml

```diff
@@ -32,9 +32,17 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: 3.11
-  
+
     - name: install nox
       run: python -m pip install nox
 
+    - name: install graphviz
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt-get install graphviz
+
+    - name: install graphviz
+      if: matrix.os == 'macos-latest'
+      run: brew install graphviz
+
```

Comment:
Why are we requiring the user to have graphviz installed locally (and installing it manually in the ci)? Wouldn't adding clingraph as a dependency to setup.cfg be cleaner, or am I missing something?

Reply:
Can clingraph produce the DOT source code for the graph without a local installation of graphviz? My understanding is that the graphviz python package can do so, and the installation of the graphviz binary is only necessary to actually render the DOT into images.

If clingraph can do that, then I guess we could just test that the DOT file that's generated matches the expected one.

Reply:
It can, but then we would still leave out from the coverage the part that renders. right?

Reply:
In the end, the nice thing of the feature is to see the image. If we want to test the DOT then we would need to do that with the visualization instead. If I understand what you mean

Reply:
Yeah that's the idea, just test that the DOT graph produced by clingraph matches the expectation (we can genereate the DOT manually and save it to file for comparison). This way we can at least test that the graph to be visualized has the correct nodes and edges etc. Of course this doesn't test all of the extra visualization stuff of clingraph. Or one could test the set of graph facts produced by clingraph.

Or just don't test it! That seems like the simplest thing to do, I'm not sure that testing the graph really adds much value either.